s29986-PrzemyslawKosiedowski/zajecia1/instalacja_debian_linux.txt

```diff
@@ -0,0 +1,1345 @@
+operacyjnego ChromeOS. 
+
+EFI System Partition - partycja ESP, niezbędna dla systemów uruchamianych w
+trybie UEFI. 
+
+physical volume for encryption - partycja, która będzie używana jako wolumin
+do szyfrowania danych. 
+
+physical volume for RAID - partycja, która będzie używana jako wolumin w
+macierzy RAID. 
+
+physical volume for LVM - partycja, która będzie używana jako wolumin dla LVM. 
+
+do no use the partition - partycja nie będzie używana ani montowana w systemie
+operacyjnym. 
+
+Strona z porównaniem systemów plików:
+https://en.wikipedia.org/wiki/Comparison_of_file_systems 
+
+MOUNT POINT – punkt montowania dla partycji: 
+
+/ - główny katalog systemowy 
+
+/boot - katalog zawierający niezbędne pliki do rozruchu systemu. 
+
+/home - katalogi domowe użytkownika. 
+
+/tmp - pliki tymczasowe. 
+
+/usr - zawiera większość programów, bibliotek i plików danych związanych z
+użytkownikiem. 
+
+/var - zawiera wszystkie zmieniające się dane, np. logi 
+
+/srv - zawiera dane serwerowe, np. pliki FTP 
+
+/opt - zawiera dodatkowe oprogramowanie, zwykle te niedostarczane przez system
+operacyjny. 
+
+/usr/local - przeznaczony dla ręcznie zainstalowanego oprogramowania
+użytkownika, które nie jest zarządzanie przez menedżera pakietów systemowych. 
+
+Enter manually – pozwala wprowadzić punkt ręcznie, podając ścieżkę dostępu do
+konkretnego katalogu na dysku. 
+
+Do not mount it – pozwala nie montować partycji w systemie; nie będzie
+dostępna do odczytu i zapisu. 
+
+MOUNT OPTIONS – opcje montowania: 
+
+discard – informuje system plików, aby natychmiast usuwał niepotrzebne bloki
+danych z urządzenia w trakcie operacji zapisu; może poprawić wydajność przy
+użyciu dysku SSD. 
+
+noatime - wyłącza aktualizację czasu ostatniego dostępu do plików. Może
+usprawnić działanie systemu. 
+
+nodiratime – robi to samo, ale wyłącznie dla katalogów. 
+
+relatime - aktualizuje czas ostatniego dostępu do plików tylko wtedy, gdy jest
+to konieczne do utrzymania poprawnego działania niektórych aplikacji. 
+
+nodev - uniemożliwia uruchomienie plików urządzeń na partycji, np. dysków czy
+kart sieciowych. 
+
+nosuid – blokuje programom możliwość korzystania ze specjalnych ustawień
+plików, które pozwalają im uzyskać dodatkowe uprawnienia. 
+
+noexec - uniemożliwia uruchomienie plików wykonywalnych na partycji. 
+
+ro – partycja jest montowana w trybie tylko do odczytu (read only).  
+
+sync – wymusza na systemie oczekiwanie, aż dane zostaną fizycznie zapisane na
+dysku przez kontynuacją działania. 
+
+usrquota - pozwala na kontrolę ilości używanej przestrzeni dyskowej przez
+poszczególnych użytkowników poprzez ustawienie limitów. 
+
+grpquota – robi to samo, ale dla grupy. 
+
+user_xattr – pozwala na przypisanie dodatkowych informacji do plików i
+katalogów poza standardowymi metadanymi, np. opis pliku. 
+
+LABEL – ustawia etykietę dla partycji. 
+
+RESERVED BLOCKS – obszar na partycji, który jest zarezerwowany do użytku przez
+system operacyjny.  
```

Comment:
Nie system operacyjny tylko dla użytkownika `root`. Jeśli na dysku nie ma miejsca a loguje się użytkownik do sesji graficznej to podczas uruchamiania sesji graficznej token łączący klienta xserwera z xserwerem musi zostać zapisany na dysk. Jeśli nie ma miejsca to nie zostanie zapisany więc sesja nigdy nie wystartuje. By zapobiec sytuacji, że nawet root nie będzie w stanie nic tu zdziałać/naprawić jest specjalna ilość miejsca zarezerwowana tylko dla roota.